### ISSUE TITLE
Reference db - grafo

### ISSUE THREAD

**Issue by freerafiki** [Lychfindel/v4w_website#34]
Riusciamo ad avere un collegamento univoco tra grafo e db? Ogni nodo nel grafo potrebbe avere un ID o un modo per essere ricollegato a un elemento del db, e la stessa cosa vale per gli archi. Ma al momento le cose sono diverse e non sembra cosi facile..

**Comment by Lychfindel** [comment]
Direi che è possibile rifacendo il grafo (e il db). Attualmente creiamo il grafo aggiungendo un nodo per ogni civico, penso sia quindi possibile dargli un ID univoco collegato al civico. Per dare un ID archi-db dovremmo ricreare gli archi del grafo partendo dal file dei toponimi (a meno che il file EL_STR non abbia già id univoci che colleghino archi e toponimi). Insomma secondo me fattibile ma lungo.

**Comment by Lychfindel** [comment]
La parte degli archi di terra è fatta.
La parte dei civici è da verificare.
La parte degli archi di acqua è da verificare.

**Comment by freerafiki** [comment]
da verificare si intende che l'id non siamo sicuri se funzioni?

**Comment by Lychfindel** [comment]
No, intendevo che non so se per il grafo acqueo ci siano gli id.
Per i civici effettivamente non abbiamo un grafo, le informazioni le prendiamo direttamente dal db
